### PR TITLE
chore: add Cirrus CI

### PR DESCRIPTION
**Pull request by RDIL**

### CODE REVIEW

File: ci-services/cirrus.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+module.exports = {
+  // The GitHub repo slug
+  repoSlug: env.CIRRUS_REPO_FULL_NAME,
+  // The name of the current branch
+  branchName: env.CIRRUS_BRANCH,
+  // Is this a regular build
+  correctBuild: env.CIRRUS_PR == null,
+  // Should the lockfile be uploaded from this build
+  uploadBuild: env.CI == 'true'
```
Comment (fkorotkov) [inline ci-services/cirrus.js]:
```suggestion
  uploadBuild: env.CI === 'true'
```

File: ci-services/cirrus.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+const env = process.env
+
+module.exports = {
+  // The GitHub repo slug
+  repoSlug: env.CIRRUS_REPO_FULL_NAME,
+  // The name of the current branch
+  branchName: env.CIRRUS_BRANCH,
+  // Is this a regular build
+  correctBuild: env.CIRRUS_PR == null,
```
Comment (fkorotkov) [inline ci-services/cirrus.js]:
```suggestion
  correctBuild: !env.CIRRUS_PR,
```

**Comment by baer95** [comment]
You might also want to add Cirrus CI to `ci-services/tests.js`:

```
...
cirrus: () => env.CIRRUS_CI === 'true',
...
```

**Comment by hulkoba** [comment]
thank you @RDIL @fkorotkov and @baer95

**Pull request merged**